marketplace/core/types/urls.py

```diff
@@ -6,19 +6,32 @@
 from marketplace.core.types.channels.generic.views import GetIcons
 from marketplace.core.types.channels.generic.views import GenericAppTypes
 
+from marketplace.core.types.externals.generic.views import (
```

Comment:
The main idea of this listing of urls is that it was generic, that way you are creating a dependency with a specific apptype, the generic-external.

The question is: if a new generic apptype comes up, will you need to implement new urls for it in this file? this deviates a little from the principles of Integrations!

I suggest that you create a way to create urls within the type itself, so that the types would be in fact generic.